organizations/data.py

```diff
@@ -168,6 +168,22 @@ def fetch_organization(organization_id):
     return organizations[0]
 
 
+def fetch_organization_by_short_name(organization_short_name):
+    """
+    Retrieves a specific organization from app/local state by short name
+    Returns a dictionary representation of the object
+    """
+    organization = {'short_name': organization_short_name}
+    if not organization_short_name:
```

Comment:
It seems to me that this validation check should come before we attempt to perform any work with the parameter of concern, so I would relocate it to the first line of the function.